### PR TITLE
docs: Removing Pattern

### PR DESCRIPTION
**Pull request by arnaudgillard**
Closes #

{New "Removing" pattern submission.}

#### Changelog

**New**

- {Images and textual content}

**Changed**

- {Navigation}

**Removed**

- {{removed thing}}

### CODE REVIEW

**Comment by alisonjoseph** [comment]
not stale

File: src/content/experimental/removing-pattern/removing.mdx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+</ImageComponent>
+
+
+## Low impact
+As removing does not destroy an asset, typically user confirmation is not required. 
+
+
+## High impact
+
+As removing does not destroy an asset, typically user confirmation is not required.
```
Comment (jnm2377) [inline src/content/experimental/removing-pattern/removing.mdx]: I think this was copied over on accident. Looks good to me once it's removed.

Comment (arnaudgillard) [reply src/content/experimental/removing-pattern/removing.mdx]: Just made the changes

**Comment by arnaudgillard** [comment]
Just updated the page :)

**Comment by vikkipaterson** [comment]
@jnm2377 is this awaiting action from our end?  (Arnaud is leaving so I am trying to work out what things are outstanding from our side to getting this completed :o)

**Comment by connor-leech** [comment]
Closing due to inactivity. This is blocked by a lack of dedicated visual designers to polish/approve. Will be reopened when picked up.

Ref: https://github.com/carbon-design-system/carbon-website/issues/1859

**Pull request closed without merging**